.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,34 +1,13 @@
-<!--
-See http://loopback.io/doc/en/contrib/triaging-pull-requests.html to understand
-the MINIMUM requirements for submitting pull requests.
--->
+### Description
 
-### Type of change
-
-<!--
-See http://semver.org/ for more info.
--->
-
-- [ ] SEMVER-MAJOR -- Backwards-incompatible changes
-- [ ] SEMVER-MINOR -- Add functionality in a backwards-compatible manner
-- [ ] SEMVER-PATCH -- Backwards-compatible bug fixes
-
-### Description of change(s)
-
-
-
-### Link(s) to related GitHub issues
 
+#### Related issues (ie. strongloop/loopback#49, etc)
 
+- None
 
 ### Checklist
 
 - [ ] CLA signed
-- [ ] Commit message conforms with the [Git commit message
-  guidelines](http://loopback.io/doc/en/contrib/git-commit-messages.html)
-- [ ] New tests are added to cover all changes
-- [ ] Code passes `npm test`
```

Comment:
A second note regarding the above is this is better for the longer term too. IMO, we do not need more bullet point items, we need to open up CI to allow community to check themselves -- the CI will tell them their commit message does not conform, etc. Just like how it tells us internally.